### DISCUSSION TITLE
Stored, persistent XSS

### DISCUSSION THREAD

**Discussion by aoprea1982** [activeadmin/activeadmin#8306] [Ideas]
Thank you for your support.

When pentesting my client activeadmin application
found a stored XSS in my context, unfiltered input.
"user-controlled input is directly interpolated into the HTML without any sanitization" in the activeadmin code.

Is ok to report it to https://tidelift.com/docs/security ? "[security@tidelift.com](mailto:security@tidelift.com)"

maybe the problem can get a CVE assigned? 

activeadmin "activeadmin-2.13.1"

Thank you!

**Comment by aoprea1982** [comment]
Found it here
[Security contact information
Please use the Tidelift security contact to [report a security vulnerability](https://tidelift.com/security). Tidelift will coordinate the fix and disclosure.](https://github.com/activeadmin/activeadmin?tab=readme-ov-file#security-contact-information)

Apologize.

Security contact information
Please use the Tidelift security contact to [report a security vulnerability](https://tidelift.com/security). Tidelift will coordinate the fix and disclosure.

**Comment by deivid-rodriguez** [comment]
[Advisory](https://github.com/activeadmin/activeadmin/security/advisories/GHSA-9mg6-x45v-hcfm) is out, thanks for the reporting the through recommended security channels :) I credited you as reporter!

**Reply by aoprea1982** [reply]
Thank you for contacting! 

Appreciate your help. 

Was busy with my family and did not notice your comments. 

One request, if possible

```
there is no mention of me in the "Credit section or "Impact"! Can that be added?
https://github.com/activeadmin/activeadmin/security/advisories/GHSA-9mg6-x45v-hcfm
```

Thank you!

**Reply by deivid-rodriguez** [reply]
Hi @aoprea1982, thanks to you!

You user name is visible in the "Credits" section (see right sidebar in the attached capture), I think that's the proper place for Credits! :)

<img width="1283" alt="Captura de pantalla 2024-06-04 a las 20 47 26" src="https://github.com/activeadmin/activeadmin/assets/2887858/6b1d0dc6-a02f-47bb-be29-f6c52fab5b41">
